packages/o2-blocks/package.json

```
@@ -0,0 +1,34 @@
+{
+	"name": "@automattic/o2-blocks",
+	"version": "1.0.0",
+	"description": "Gutenberg extensions for o2 theme.",
+	"main": "dist/editor.js",
+	"sideEffects": false,
+	"repository": {
+		"type": "git",
+		"url": "git://github.com/Automattic/wp-calypso.git",
+		"directory": "packages/o2-blocks"
+	},
+	"publishConfig": {
+		"access": "public"
+	},
+	"author": "Automattic, Inc.",
+	"license": "GPL-2.0-or-later",
+	"bugs": {
+		"url": "https://github.com/Automattic/wp-calypso/issues"
+	},
+	"homepage": "https://github.com/Automattic/wp-calypso",
+	"scripts": {
+		"clean": "npx rimraf dist",
+		"prepublish": "npm run clean",
+		"prepare": "npx webpack --config ./webpack.config.js"
```

Ideally:

```suggestion
		"prepare": "npx webpack editor=./src/editor.js --output-path=./dist"
```

which would allow us to remove `webpack.config.js`. But that fails with `Can't resolve 'src/editor.js' in '/home/bernie/src/wp-calypso/packages/calypso-build'` (apparently webpack tries to resolve the entry point path relative to the location of the `webpack.config.js` it uses.

We might be able to circumvent this by adding a `bin` alias to `@automattic/calypso-build/package.json` -- but that also means updating all current to what is currently the `@automattic/calypso-build` binary (`bin/build-package.js`) -- som let's maybe do that in a follow-up PR.

---

Needing 20 lines of Webpack isn't too tragic for this package. :-)

---

Need to remove the [`context`](https://webpack.js.org/configuration/entry-context/) 🙂 

https://github.com/Automattic/wp-calypso/pull/32045